README.md

```diff
@@ -28,31 +28,134 @@ flag is highly recommended for those new to [pip](https://pip.pypa.io).
 A PEX executable (Windows not supported) and source releases are
 [here](https://github.com/planetlabs/planet-client-python/releases/latest).
 
-## Documentation
+## API Key
 
-Online documentation:
-https://planetlabs.github.io/planet-client-python/index.html
+The planet API requires an account for use. [Signup here](https://www.planet.com/explorer/?signup).
 
-Documentation is also provided for download
-[here](https://github.com/planetlabs/planet-client-python/releases/latest).
+This can be provided via the environment variable `PL_API_KEY` or the flag `-k` or `--api-key`.
 
+Using `planet init` your account credentials (login/password) can be used to obtain the api key.
 
-## Development
+## Quick Start
 
-To contribute or develop with this library, see
-[CONTRIBUTING](https://github.com/planetlabs/planet-client-python/CONTRIBUTING.md)
+### Python Library
 
+The python library is asynchronous, which greatly speeds up many interactions
+with the planet API. 
 
-## API Key
+```python
+import asyncio
+import os
+from planet.api import APlanetSession, AOrdersClient
+from planet.api.order_details import OrderDetails, Product
 
-The API requires an account for use. [Signup here](https://www.planet.com/explorer/?signup).
+API_KEY = os.getenv('PL_API_KEY')
 
-This can be provided via the environment variable `PL_API_KEY` or the flag `-k` or `--api-key`.
+image_ids = ['3949357_1454705_2020-12-01_241c']
+order_detail = OrderDetails(
+    'test_order',
+    [Product(image_ids, 'analytic', 'psorthotile')]
+)
 
-Using `planet init` your account credentials (login/password) can be used to obtain the api key.
+async def create_order(order_detail):
```

Comment:
This seems like the sort of thing that might be nice for a batteries-included experience, e.g.

```python
import asyncio

import planet

order = {
    "name":
    "test_order",
    "products": [{
        "image_ids": "3949357_1454705_2020",
        "item_type": "psorthotile",
        "product_bundle": "analytic"
    }]
}

oid = asyncio.run(planet.create_order(planet.get_api_key(), order))
print(oid)
```

Reply:
Yes! You are right. This was implemented only in the CLI in v1, I agree it makes total sense to include this in the python lib.

Reply:
https://github.com/planetlabs/planet-client-python/issues/248

Reply:
Would love additional discussion in #248. For now, for this PR, I'll mark this as resolved.